Packs/MalwareInvestigationAndResponse/README.md

```diff
@@ -21,7 +21,7 @@ Tagging of malicious and benign indicators to tie into allow & deny lists for th
 Incident mirroring & closure with EDR
 Support for aggregate incidents from EDRs, which may include multiple alerts & files
 
-![Malware Investigation & Response Incident Handler](https://raw.githubusercontent.com/demisto/content/78e4fb246c3fa57e6201ee2791fd4919864ed956/Packs/MalwareInvestigationAndResponse/doc_files/Malware_Investigation_%26_Response_Incident_Handler.png)
+![Malware Investigation & Response Incident layout](./../../doc_files/Malware_Investigation_&_Response_layout.png)
```

Comment:
```suggestion
![Malware Investigation & Response Incident layout](./doc_files/Malware_Investigation_&_Response_layout.png)
```

Reply:
Relative URLs are not supported in the pack readme, used absolute URLs instead.
https://app.circleci.com/pipelines/github/demisto/content/167335/workflows/fa548c75-63eb-4333-ae8c-f83c035bfd71/jobs/525463